static/js/resize.js

```diff
@@ -42,8 +42,12 @@ function get_new_heights() {
     const res = {};
     const viewport_height = message_viewport.height();
     const top_navbar_height = $("#top_navbar").safeOuterHeight(true);
+    const navbar_alerts_height = $("#panels").safeOuterHeight(true) || 0;
     const invite_user_link_height = $("#invite-user-link").safeOuterHeight(true) || 0;
-    const add_streams_link_height = $("#add-stream-link").safeOuterHeight(true) || 0;
+
+    // Change add_stream_link height by adding navbar_alerts height
+    const add_streams_link_height =
+        ($("#add-stream-link").safeOuterHeight(true) || 0) + navbar_alerts_height;
```

Comment:
`message_viewport.height()` already excludes `#panels`, so this will overcorrect as soon as the window is resized.

Reply:
@andersk I observed that until close the navbar alerts(#pane) it won't overcorrect. I will check again.